### ISSUE TITLE
Interpolate `AbstractSolution`  

### ISSUE THREAD

**Issue by mvanzulli** [mvanzulli/ONSAS.jl#136]
Create a way to interpolate the `AbstractSolution` struct. 

Similar to Ferrite.jl we can create a `PointEvalHandler` that might be used like this:
```julia
ph = PointEvalHandler(mesh::AbstractMesh, vec_points::Vector{<:Vector{<:Real}})
```
The `PointEvalHandler` object creates the interpolation matrix related to the element shape functions defined in the `mesh`. 
Finally the solution is obtained via:
```julia
u_sol = displacements(sol::AbstractSolution, ph::PointEvalHandler)::Vector{Vector{<:Real}}
```
This point handler can be useful to generate .`vtk` files.

**Comment by mvanzulli** [comment]
I think at this point we might consider to add shape functions for the elements and the use dispatch, or maybe an `interpolate` method that each element should implement.

**Comment by mvanzulli** [comment]
Closed via #142
